openslides/motions/serializers.py

```diff
@@ -337,6 +290,37 @@ def validate(self, data):
         return data
 
 
+class MotionCommentFieldSerializer(ModelSerializer):
+    """
+    Serializer for motion.models.MotionCommentField objects.
+    """
+    class Meta:
+        model = MotionCommentField
+        fields = (
+            'id',
+            'name',
+            'read_groups',
+            'write_groups',)
+
+
+class MotionCommentSerializer(ModelSerializer):
+    """
+    Serializer for motion.models.MotionComment objects.
+    """
+    read_groups_id = SerializerMethodField()
+
+    class Meta:
+        model = MotionComment
+        fields = (
+            'id',
+            'comment',
+            'field',
+            'read_groups_id',)
+
+    def get_read_groups_id(self, comment):
+        return [group.id for group in comment.field.read_groups.all()]
```

Comment:
This causes extra db queries so we need a special model manager (and some query tests).

Reply:
Added a comment manager. Also updated `test_motion_db_queries`. Works fine.